konig-transform/src/test/resources/konig-transform/aurora-transform/shapes/shape_TargetPersonShape.ttl

```diff
@@ -1,33 +1,82 @@
-@prefix as: <http://www.w3.org/ns/activitystreams#> .
-@prefix aws: <http://www.konig.io/ns/aws/> .
-@prefix konig: <http://www.konig.io/ns/core/> .
-@prefix prov: <http://www.w3.org/ns/prov#> .
-@prefix schema: <http://schema.org/> .
-@prefix sh: <http://www.w3.org/ns/shacl#> .
-@prefix shape: <http://example.com/shapes/> .
-@prefix xsd: <http://www.w3.org/2001/XMLSchema#> .
-
-shape:TargetPersonShape a sh:Shape , konig:TargetShape; 
-	prov:wasGeneratedBy <http://www.konig.io/activity/hSbp2OqGW5gAAAFiAfnJCw> ; 
-	sh:targetClass schema:Person ; 
-	sh:nodeKind sh:IRI ; 
-	sh:property  [ 
-		sh:path schema:givenName ; 
-		sh:datatype xsd:string ; 
-		sh:minCount 0 ; 
-		sh:maxCount 1
-	 ]  ,  [ 
-		sh:path schema:familyName ; 
-		sh:datatype xsd:string ; 
-		sh:minCount 0 ; 
-		sh:maxCount 1 ]  ; 
-	konig:shapeDataSource <http://www.konig.io/ns/aws/host/${awsAuroraHost}/databases/${awsAuroraSchema}/tables/TargetPersonShape> . 
-
-<http://www.konig.io/activity/hSbp2OqGW5gAAAFiAfnJCw> a konig:LoadModelFromSpreadsheet ; 
-	as:endTime "2018-03-07T14:39:30.044-05:00"^^xsd:dateTime . 
-
-<http://www.konig.io/ns/aws/host/${awsAuroraHost}/databases/${awsAuroraSchema}/tables/TargetPersonShape> a konig:AwsAurora , konig:DataSource ; 
-	aws:tableReference  [ 
-		aws:awsAuroraHost "${awsAuroraHost}" ; 
-		aws:awsSchema "${awsAuroraSchema}" ; 
+@prefix as: <http://www.w3.org/ns/activitystreams#> .
+@prefix aws: <http://www.konig.io/ns/aws/> .
+@prefix konig: <http://www.konig.io/ns/core/> .
+@prefix prov: <http://www.w3.org/ns/prov#> .
+@prefix schema: <http://schema.org/> .
+@prefix sh: <http://www.w3.org/ns/shacl#> .
+@prefix shape: <http://example.com/shapes/> .
+@prefix xsd: <http://www.w3.org/2001/XMLSchema#> .
+
+shape:TargetPersonShape a sh:Shape , konig:TargetShape ; 
+	prov:wasGeneratedBy <http://www.konig.io/activity/wCgoIXFswSIAAAFidlxSfQ> ; 
+	sh:targetClass schema:Person ; 
+	konig:inputShapeOf konig:TargetShape ; 
```

Comment:
The value "konig:TargetShape" will work for testing purposes, but it is not an appropriate value.  Please change to name of a fictitious system component such as "ex:MasterDataEtl"  where "ex" is the the prefix for an example namespace -- "http://example.com/systems/"